src/kmer.py

```diff
@@ -7,25 +7,41 @@ def kmer(x: str, k: int) -> list[str]:
 
     >>> kmer('agtagtcg', 3)
     ['agt', 'gta', 'tag', 'agt', 'gtc', 'tcg']
-
-    FIXME: do you want more tests here?
     """
-    ...
+    kmers=[]
+    for i in range(0, len(x)-k+1):
+        kmerr=x[i:i+k]
+        kmers.append(kmerr)
+    return kmers
+
 
 
 def unique_kmers(x: str, k: int) -> list[str]:
     """
-    Computer all unique k-mers of x.
-
-    FIXME: do you want more tests here?
+    >>> unique_kmers('abcd', 2)
+    ['ab', 'bc', 'cd']
     """
-    ...
+    kmers=[]
+    for i in range(0, len(x)-k+1):
+        kmerr=x[i:i+k]
+        if kmerr not in kmers:
+            kmers.append(kmerr)
+    return kmers
 
 
 def count_kmers(x: str, k: int) -> dict[str, int]:
     """
-    Computer all k-mers of x and count how often they appear.
-
-    FIXME: do you want more tests here?
+    >>> count_kmers('abcd', 4)
+    {'abcd': 1}
     """
-    ...
+    kmer_count={}
+    kmers=[]
+    for i in range(0, len(x)-k+1):
+        kmerr=x[i:i+k]
+        kmers.append(kmerr)
```

Comment:
You are repeating the code from another function. Just call the function instead. It's easier.